code/modules/mob/living/carbon/brain/life.dm

```diff
@@ -33,20 +33,21 @@
 				updatehealth()
 
 /mob/living/carbon/brain/handle_environment(datum/gas_mixture/environment)
-	if(!environment)
-		return
-	var/environment_heat_capacity = environment.heat_capacity()
-	if(istype(get_turf(src), /turf/space))
-		var/turf/heat_turf = get_turf(src)
-		environment_heat_capacity = heat_turf.heat_capacity
+	if(!istype(src, /obj/item/device/mmi/posibrain)) //we don't want posibrains suffocating, now do we?

```

Comment:
Read off the code for the surgery step for removing the brain, and apply it to yourself.